### DISCUSSION TITLE
What is the char for the lock glyph?

### DISCUSSION THREAD

**Discussion by Moosems** [tonsky/FiraCode#1540] [Q&A] [answered]
There's a lock character in the image for Fira Code.

**Comment by alexeyten** [comment]
Code point | Glyph | Description
-- | -- | --
U+E0A0 | `` | Version control branch
U+E0A1 | `` | LN (line) symbol
U+E0A2 | `` | Closed padlock
U+E0B0 | `` | Rightwards black arrowhead
U+E0B1 | `` | Rightwards arrowhead
U+E0B2 | `` | Leftwards black arrowhead
U+E0B3 | `` | Leftwards arrowhead

**Reply by Moosems** [reply]
Where can I find a list of all chars and their description?

**Reply by alexeyten** [reply, marked as answer]
See https://github.com/ryanoasis/powerline-extra-symbols#glyphs
Only first row has a wide support, other fancy symbols (like “flame”) may be or may be not available in fonts.

Also you could check https://www.nerdfonts.com/.

**Reply by Moosems** [reply]
I'm pretty much only using Fira Code.
